server/config/requestHandler.js

```diff
@@ -49,4 +49,18 @@ module.exports = {
     res.send(responseJSON);
   },
 
-};
\ No newline at end of file
+  getPhotos: function(req, res) {
+    var responseJSON = {
+      photos: Photos
```

Comment:
We are we sending back the entire photos array here? As with my other comment, I'm probably just misunderstanding what you're trying to do.